src/pages/Introduction.css

```diff
@@ -84,3 +88,44 @@
   padding: 10vh 10vw;
   background-image: radial-gradient(ellipse farthest-corner at center, rgba(255,255,255,0.88) 0%, rgba(255,255,255,0) 75%);
 }
+
+.starButton {
```

Comment:
The start button would make a great candidate for a standalone component. 
If it should just be here than please add `Intro-` as a prefix so that it does not conflict with the CSS of other elements.